packages/contracts/src/contracts/current/protocol/Exchange/mixins/MSignatureValidator.sol

```diff
@@ -32,6 +32,7 @@ contract MSignatureValidator is
         EIP712,
         Trezor,
         Contract,
+        Validator,
```

Comment:
Hm... Since Validator is most likely also a smart contract, the naming here is not ideal. What about `MakerContract` and `NonMakerContract`?

Reply:
I don't love these names either...

Reply:
Maybe `ContractOwner` and `ContractDelegate`? Don 't love `maker` here, since you can use these sig types for other things too (sender abstraction)

Reply:
Yeah, perhaps `signer` and `validator`.